Rollbar/NetStandard/RollbarConfigurationLoader.cs

```diff
@@ -0,0 +1,138 @@
+﻿namespace Rollbar.NetStandard
+{
+    using System;
+    using System.Diagnostics;
+    using System.IO;
+    using System.Reflection;
+
+    using Rollbar.Common;
+    using Rollbar.Telemetry;
+
+    /// <summary>
+    /// Class RollbarConfigurationLoader.
+    /// Implements the <see cref="Rollbar.NetStandard.IRollbarConfigurationLoader" />
+    /// </summary>
+    /// <seealso cref="Rollbar.NetStandard.IRollbarConfigurationLoader" />
+    public class RollbarConfigurationLoader 
+        : IRollbarConfigurationLoader
+    {
+        /// <summary>
+        /// The trace source
+        /// </summary>
+        private static readonly TraceSource traceSource = new TraceSource(typeof(RollbarConfigurationLoader).FullName);
+
+        /// <summary>
+        /// The known assembly names
+        /// </summary>
+        private readonly static string[] knownAssemblyNames = {
+            "Rollbar.App.Config",
+            "Rollbar.AppSettings.Json",
+            };
```

Comment:
Nit: I think that terminal brace should shift left 4 spaces.

Reply:
Not according to the Visual Studio's  code formatter for array initialization :). 
Since we do not have explicit code style spec and mainly rely on MS recommended .NET coding style guided by the VS let's leave it as is.